website/docs/d/workspace.html.markdown

```diff
@@ -30,8 +30,8 @@ The following arguments are supported:
 
 In addition to all arguments above, the following attributes are exported:
 
-* `id` - The workspace's human-readable ID, which looks like
-  `<ORGANIZATION>/<WORKSPACE>`.
+* `id` - The workspace's opaque external ID, which looks like
+  `ws-<RANDOM STRING>`.
 * `external_id` - The workspace's opaque external ID, which looks like
```

Comment:
This one should be removed right? No point in exporting the same ID twice with a different name.